excluded_nodes: ['/rosout']

# You should also exclude the topics produced by an excluded node!
excluded_topics: ['/rosout', '/rosout_agg']